### ISSUE TITLE
Changing FlowColumnCount does not udpate the control

### ISSUE THREAD

**Issue by Zure1** [daniel-luberda/DLToolkit.Forms.Controls#79]
I'm trying to change `FlowColumnCount` when the event `OnSizeAllocated` is fired:

```
protected override void OnSizeAllocated(double width, double height)
{
    Device.BeginInvokeOnMainThread(() =>
    {
        base.OnSizeAllocated(width, height); // must be called
        if (this.width != width || this.height != height)
        {
            this.width = width;
            this.height = height;
            flowListViewWidgets.FlowColumnCount = height > width ? 2 : 3;
            flowListViewWidgets.BackgroundColor = flowListViewWidgets.BackgroundColor != Color.Aqua ? Color.Aqua : Color.Red;
        }
    });
}
```

If the device is in portraitmode, `FlowColumnCount` is set to 2. In landscape mode, it's set to 3. However, this has no effect to the FlowListView! The controls inside the `DataTemplate` will not be re-located according to the new `FlowColumnCount`.

Portrait (FlowColumnCount = 2, BackgroundColor = Aqua):

![portrait](https://cloud.githubusercontent.com/assets/22812225/23798756/3f48d6e4-05a6-11e7-8d46-52ddef00d115.png)

Landscape (FlowColumnCount = 3, BackgroundColor = Red):

![landscape](https://cloud.githubusercontent.com/assets/22812225/23798755/3f47b1e2-05a6-11e7-9d2d-cbbabc291dda.png)

I'm changing the `BackgroundColor` just to prove that the BackgroundColor does update when I set this value, but for `FlowColumnCount`, it does not.

**Comment by Zure1** [comment]
@daniel-luberda After taking a look into your code, I noticed that `DesiredColumnCount` is actually the value that defines the behaviour. When setting `FlowColumnCount`, the value for `DesiredColumnCount` is not updated at the moment. A workaround is to call `FlowListView.ForceReload()` because this calls `RefreshDesiredColumnCount()`. But i think that when I set `FlowColumnCount`, it should automatically refresh the value for `DesiredColumnCount` too.

And btw, when I first call `ForceRefresh()`, I'm getting a `NullReferenceException`. My flowListView ist not null tho.

Thanks for this control Daniel, I really love it!
